soccer/planning/low_level/Replanner.hpp

```diff
@@ -0,0 +1,54 @@
+#pragma once
```

Comment:
TODO make these free functions (maybe - that might not interact nicely with the config system?)